src/helpers/figures/charts/chart_common_line_scatter.ts

```diff
@@ -179,7 +179,7 @@ export function getTrendDatasetForLineChart(
   if (!newValues.length) {
     return;
   }
-  return getFullTrendingLineDataSet(dataset, config, newValues);
+  return getFullTrendingLineDataSet(dataset, config, newValues, newLabels);
```

Comment:
nitpick: variables names are a bit confusing IMO. Something like `trendLabels` and `trendValues` is more correct no ?